benchmark/manifests/BenchmarkK8sService.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: benchmark-tool
+  name: benchmark-tool
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: benchmark-tool
+  template:
+    metadata:
+      labels:
+        app: benchmark-tool
+    spec:
+      containers:
+      - image: 'us-docker.pkg.dev/cloud-tpu-images/inference/inference-benchmark@sha256:1c100b0cc949c7df7a2db814ae349c790f034b4b373aaad145e77e815e838438'
```

Comment:
What is this inference benchmark tool? Is there a way to share more / link to what this is for any further explanation of parameters and options?

Reply:
It's the LPG tool explained in the begging of the guide. I also added a link to the commit that the image is based on